valexa/core/profiles.py

```diff
@@ -8,15 +8,17 @@
 import math
 import numpy as np
 
+import io
+
 from valexa.core.standard import Standard
 from valexa.core.models import Result, ModelHandler, Model
 
 DEFAULT_TOLERANCE = 80
-DEFAULT_ACCEPTANCE = 20
+DEFAULT_ACCEPTANCE = 50
 
 
 def make_profiles(calib_data: List[tuple], valid_data: List[tuple], tolerance_limit: int,
-                  acceptance_limit: int) -> List:
+                  acceptance_limit: int, file_name: str) -> List:
```

Comment:
What is the purpose of the filename here?

Reply:
file_name is used in the creation of the report where it put the file_name in the header of the graph. See core/html.py line 30.